docs/index.md

```diff
@@ -0,0 +1,127 @@
+---
+repository: "https://github.com/turbot/steampipe-mod-github-compliance"
+---
+
+# GitHub Compliance Mod
+
+Run security controls across all your AWS accounts to look for resources that are publicly accessible resources, shared with untrusted accounts, have insecure network configurations, and more.
```

Comment:
```suggestion
Run security controls across all your GitHub repositories to look for resources that are publicly accessible resources, shared with untrusted accounts, have insecure network configurations, and more.
```
The description should be updated to use the format of GitHub resources (repositories and organizatons).